GrobidMetadataExtractorPlugin.php

```diff
@@ -106,7 +107,9 @@ function editSubmissionFile(string $hookName, array $args) : bool
             $author->setData('email', '');
             $author->setData('givenName', $xpath->query('tei:persName/tei:forename[@type="first"]', $authorNode)->item(0)->nodeValue, $primaryLocale);
             $author->setData('familyName', $xpath->query('tei:persName/tei:surname', $authorNode)->item(0)->nodeValue, $primaryLocale);
-            $author->setData('affiliation', $xpath->query('tei:affiliation/tei:orgName', $authorNode)->item(0)->nodeValue, $primaryLocale);
+            $affiliation = new Affiliation();
+            $affiliation->setName($xpath->query('tei:affiliation/tei:orgName', $authorNode)->item(0)->nodeValue, $primaryLocale);
```

Comment:
Do we like to look for RORs here? Similar to here: https://github.com/bozana/pkp-lib/blob/7135/plugins/importexport/native/filter/NativeXmlPKPAuthorFilter.php#L117-L129.
I.e. something like:
$name = $xpath->query('tei:affiliation/tei:orgName', $authorNode)->item(0)->nodeValue;
$affiliation->setName($name, $primaryLocale);
$ror = Repo::ror()->getCollector()->filterByName($name)->getMany()->first();
if ($ror) {
  $affiliation->setRor($ror->getRor());
  $affiliation->setName(null);
}

Reply:
No objections -- but this is just an experimental plugin at the moment, so consider it generally subject to review/improvement.